please/please.py

```diff
@@ -332,6 +332,8 @@ def setup() -> None:
     config["disable_greeting"] = False
     config["time_format_24h"] = False
     config["last_reminder"] = None
+    conig["done_icon"] = "✅"
```

Comment:
```suggestion
    config["done_icon"] = "✅"
```

Reply:
😅 I guess I missed this typo as I had those variables in config.
Will update it.